build/Dockerfile

```diff
@@ -6,36 +6,38 @@ RUN dpkg --add-architecture i386 \
 	&& apt-get update -y \
 	&& apt-get install -y --no-install-recommends ca-certificates curl lib32gcc1 lib32stdc++6 libcurl4-gnutls-dev:i386 \
 	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
-
-# Install gosu.
-RUN gpg --keyserver pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
+	&& rm -rf /var/lib/apt/lists/* \
+	&& gpg --keyserver pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
 	&& curl -o /usr/local/bin/gosu -fsSL "https://github.com/tianon/gosu/releases/download/1.7/gosu-$(dpkg --print-architecture)" \
 	&& curl -o /usr/local/bin/gosu.asc -fsSL "https://github.com/tianon/gosu/releases/download/1.7/gosu-$(dpkg --print-architecture).asc" \
 	&& gpg --verify /usr/local/bin/gosu.asc \
 	&& rm /usr/local/bin/gosu.asc \
 	&& chmod +x /usr/local/bin/gosu
 
+# We just copy the necessary scripts to download the game. This improves the development speed
+# because we don't need to download the game again if it's not necessary.
+COPY /script/steamcmd /script/update.sh /usr/local/bin/
```

Comment:
The files inside `/script/` change *a lot*. The game change less than this files. At list now that we are in development. And download the game takes *a lot* of time. If we copy the files after we download the game we don't need to download the game again. I can remove the first `COPY` but I'll need to use the `steamcmd.sh` command.

Any way I think that when we release `v1.0` we can merge both in the top. In that momment (I hope) the game'll change more thant those files.